<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>control</string>
    </property>
    <widget class="QMenu" name="menuState">
     <property name="title">
      <string>State</string>
     </property>
     <addaction name="actionSave_State"/>
     <addaction name="separator"/>
     <addaction name="actionLoad_State"/>
    </widget>
    <widget class="QMenu" name="menuCopy_Paste">
     <property name="title">
      <string>Copy/Paste</string>
     </property>
     <addaction name="actionPaste_from_Clipboard"/>
     <addaction name="actionStop_Pasting"/>
    </widget>
    <widget class="QMenu" name="menuCpu_Speed">
     <property name="title">
      <string>Cpu Speed</string>
     </property>
     <addaction name="actionSpeed_x1"/>
     <addaction name="actionSpeed_x2"/>
     <addaction name="actionSpeed_x4"/>
     <addaction name="actionSpeed_x8"/>
     <addaction name="actionSpeed_x16"/>
    </widget>
    <widget class="QMenu" name="menuDebugger">
     <property name="title">
      <string>Debugger</string>
     </property>
     <addaction name="actionDebugger_1"/>
     <addaction name="actionDebugger_2"/>
     <addaction name="actionDebugger_3"/>
     <addaction name="separator"/>
     <addaction name="actionClose_Debuggers"/>
    </widget>
    <addaction name="actionReset"/>
    <addaction name="actionSpecial_Reset"/>
    <addaction name="separator"/>
    <addaction name="menuCpu_Speed"/>
    <addaction name="separator"/>
    <addaction name="menuCopy_Paste"/>
    <addaction name="separator"/>
    <addaction name="menuState"/>
    <addaction name="menuDebugger"/>
    <addaction name="separator"/>
    <addaction name="actionExit_Emulator"/>
   </widget>
   <widget class="QMenu" name="menuFD0">
    <property name="title">
     <string>Floppy</string>
    </property>
    <widget class="QMenu" name="menuWrite_Protection_FD0">
     <property name="title">
      <string>Write Protection</string>
     </property>
     <addaction name="actionProtection_ON_FD0"/>
     <addaction name="actionProtection_OFF_FD0"/>
    </widget>
    <addaction name="actionInsert_FD0"/>
    <addaction name="actionEject_FD0"/>
    <addaction name="separator"/>
    <addaction name="actionRecent_Opened_FD0"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_D88_Image_FD0"/>
    <addaction name="separator"/>
    <addaction name="menuWrite_Protection_FD0"/>
   </widget>
   <widget class="QMenu" name="menuQD0">
    <property name="title">
     <string>QD</string>
    </property>
    <widget class="QMenu" name="menuWrite_Protection_QD0">
     <property name="title">
      <string>Write Protection</string>
     </property>
     <addaction name="actionProtection_ON_QD0"/>
     <addaction name="actionProtection_OFF_QD0"/>
    </widget>
    <addaction name="actionInsert_QD0"/>
    <addaction name="actionEject_QD0"/>
    <addaction name="separator"/>
    <addaction name="actionResent_Images_QD0"/>
    <addaction name="separator"/>
    <addaction name="menuWrite_Protection_QD0"/>
   </widget>
   <widget class="QMenu" name="menuCMT">
    <property name="title">
     <string>CMT</string>
    </property>
    <widget class="QMenu" name="menuWrite_Protection_3">
     <property name="title">
      <string>Write Protection</string>
     </property>
     <addaction name="actionProtection_ON_CMT"/>
     <addaction name="actionProtection_OFF_CMT"/>
    </widget>
    <addaction name="actionInsert"/>
    <addaction name="actionEject_3"/>
    <addaction name="separator"/>
    <addaction name="actionPlay_Start"/>
    <addaction name="actionPlay_Stop"/>
    <addaction name="separator"/>
    <addaction name="actionRecording"/>
    <addaction name="separator"/>
    <addaction name="menuWrite_Protection_3"/>
   </widget>
   <widget class="QMenu" name="menuScreen">
    <property name="title">
     <string>Screen</string>
    </property>
    <widget class="QMenu" name="menuStretch_Mode">
     <property name="title">
      <string>Stretch Mode</string>
     </property>
     <addaction name="actionDot_by_Dot"/>
     <addaction name="actionKeep_Aspect"/>
     <addaction name="actionFill_Display"/>
    </widget>
    <addaction name="actionZoom"/>
    <addaction name="actionDisplay_Mode"/>
    <addaction name="separator"/>
    <addaction name="menuStretch_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionScanLine"/>
    <addaction name="actionCRT_Filter"/>
   </widget>
   <widget class="QMenu" name="menuSound">
    <property name="title">
     <string>Sound</string>
    </property>
    <widget class="QMenu" name="menuOutput_Frequency">
     <property name="title">
      <string>Output Frequency</string>
     </property>
     <addaction name="action2000Hz"/>
     <addaction name="action4000Hz"/>
     <addaction name="action8000Hz"/>
     <addaction name="action11025Hz"/>
     <addaction name="action22050Hz"/>
     <addaction name="action44100Hz"/>
     <addaction name="action48000Hz"/>
     <addaction name="action96000Hz"/>
    </widget>
    <widget class="QMenu" name="menuSound_Latency">
     <property name="title">
      <string>Sound Latency</string>
     </property>
     <addaction name="action50ms"/>
     <addaction name="action100ms"/>
     <addaction name="action200ms"/>
     <addaction name="action300ms"/>
     <addaction name="action400ms"/>
    </widget>
    <addaction name="menuOutput_Frequency"/>
    <addaction name="menuSound_Latency"/>
   </widget>
   <widget class="QMenu" name="menuFrequency">
    <property name="title">
     <string>Machine</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Record</string>
    </property>
    <widget class="QMenu" name="menuRecord_sound">
     <property name="title">
      <string>Record sound</string>
     </property>
     <addaction name="actionStart_Record"/>
     <addaction name="actionStop_Record"/>
    </widget>
    <widget class="QMenu" name="menuRecoad_as_movie">
     <property name="title">
      <string>Recoad as movie</string>
     </property>
     <addaction name="actionStart_Record_Movie"/>
     <addaction name="actionStop_Record_Movie"/>
    </widget>
    <addaction name="actionCapture_Screen"/>
    <addaction name="separator"/>
    <addaction name="menuRecoad_as_movie"/>
    <addaction name="separator"/>
    <addaction name="menuRecord_sound"/>
   </widget>
   <widget class="QMenu" name="menuEmulator">
    <property name="title">
     <string>Emulator</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHELP">
    <property name="title">
     <string>HELP</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuControl"/>
   <addaction name="menuFD0"/>
   <addaction name="menuQD0"/>
   <addaction name="menuCMT"/>
   <addaction name="menuFrequency"/>
   <addaction name="menuSound"/>
   <addaction name="menuScreen"/>
   <addaction name="menuRecord"/>
   <addaction name="menuEmulator"/>
   <addaction name="menuHELP"/>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSpecial_Reset">
   <property name="text">
    <string>Special Reset</string>
   </property>
  </action>
  <action name="actionExit_Emulator">
   <property name="text">
    <string>Exit Emulator</string>
   </property>
  </action>
  <action name="actionSpeed_x1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed x1</string>
   </property>
  </action>
  <action name="actionSpeed_x2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed x2</string>
   </property>
  </action>
  <action name="actionSpeed_x4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed x4</string>
   </property>
  </action>
  <action name="actionSpeed_x8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Seppd x8</string>
   </property>
  </action>
  <action name="actionSpeed_x16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed x16</string>
   </property>
  </action>
  <action name="actionPaste_from_Clipboard">
   <property name="text">
    <string>Paste from Clipboard</string>
   </property>
  </action>
  <action name="actionStop_Pasting">
   <property name="text">
    <string>Stop Pasting</string>
   </property>
  </action>
  <action name="actionSave_State">
   <property name="text">
    <string>Save State</string>
   </property>
  </action>
  <action name="actionLoad_State">
   <property name="text">
    <string>Load State</string>
   </property>
  </action>
  <action name="actionDebugger_1">
   <property name="text">
    <string>Debugger 1</string>
   </property>
  </action>
  <action name="actionDebugger_2">
   <property name="text">
    <string>Debugger 2</string>
   </property>
  </action>
  <action name="actionDebugger_3">
   <property name="text">
    <string>Debugger 3</string>
   </property>
  </action>
  <action name="actionClose_Debuggers">
   <property name="text">
    <string>Close Debuggers</string>
   </property>
  </action>
  <action name="actionInsert_FD0">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionEject_FD0">
   <property name="text">
    <string>Eject</string>
   </property>
  </action>
  <action name="actionRecent_Opened_FD0">
   <property name="text">
    <string>Recent Opened</string>
   </property>
  </action>
  <action name="actionSelect_D88_Image_FD0">
   <property name="text">
    <string>Select D88 Image</string>
   </property>
  </action>
  <action name="actionProtection_ON_FD0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Protection ON</string>
   </property>
  </action>
  <action name="actionProtection_OFF_FD0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Protection OFF</string>
   </property>
  </action>
  <action name="actionInsert_QD0">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionEject_QD0">
   <property name="text">
    <string>Eject</string>
   </property>
  </action>
  <action name="actionResent_Images_QD0">
   <property name="text">
    <string>Resent Images</string>
   </property>
  </action>
  <action name="actionProtection_ON_QD0">
   <property name="text">
    <string>Protection ON</string>
   </property>
  </action>
  <action name="actionProtection_OFF_QD0">
   <property name="text">
    <string>Protection OFF</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionEject_3">
   <property name="text">
    <string>Eject</string>
   </property>
  </action>
  <action name="actionPlay_Start">
   <property name="text">
    <string>Play Start</string>
   </property>
  </action>
  <action name="actionPlay_Stop">
   <property name="text">
    <string>Play Stop</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="text">
    <string>Recording</string>
   </property>
  </action>
  <action name="actionProtection_ON_CMT">
   <property name="text">
    <string>Protection ON</string>
   </property>
  </action>
  <action name="actionProtection_OFF_CMT">
   <property name="text">
    <string>Protection OFF</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom Screen</string>
   </property>
  </action>
  <action name="actionDisplay_Mode">
   <property name="text">
    <string>Display Mode</string>
   </property>
  </action>
  <action name="actionScanLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set ScanLine</string>
   </property>
  </action>
  <action name="actionCRT_Filter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>CRT Filter</string>
   </property>
  </action>
  <action name="actionDot_by_Dot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dot by Dot</string>
   </property>
  </action>
  <action name="actionKeep_Aspect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep Aspect</string>
   </property>
  </action>
  <action name="actionFill_Display">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fill Display</string>
   </property>
  </action>
  <action name="actionCapture_Screen">
   <property name="text">
    <string>Capture Screen</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="action2000Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2000Hz</string>
   </property>
  </action>
  <action name="action4000Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4000Hz</string>
   </property>
  </action>
  <action name="action8000Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8000Hz</string>
   </property>
  </action>
  <action name="action11025Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>11025Hz</string>
   </property>
  </action>
  <action name="action22050Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>22050Hz</string>
   </property>
  </action>
  <action name="action44100Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>44100Hz</string>
   </property>
  </action>
  <action name="action48000Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>48000Hz</string>
   </property>
  </action>
  <action name="action96000Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>96000Hz</string>
   </property>
  </action>
  <action name="action50ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50ms</string>
   </property>
  </action>
  <action name="action100ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>100ms</string>
   </property>
  </action>
  <action name="action200ms">
   <property name="text">
    <string>200ms</string>
   </property>
  </action>
  <action name="action300ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>300ms</string>
   </property>
  </action>
  <action name="action400ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>400ms</string>
   </property>
  </action>
  <action name="actionStart_Record">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start Record</string>
   </property>
  </action>
  <action name="actionStop_Record">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop Record</string>
   </property>
  </action>
  <action name="actionStart_Record_Movie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start Record Movie</string>
   </property>
  </action>
  <action name="actionStop_Record_Movie">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop Record Movie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionCRT_Filter</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionCRT_Filter</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit_Emulator</sender>
   <signal>destroyed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProtection_OFF_FD0</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionProtection_OFF_FD0</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProtection_ON_FD0</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionProtection_ON_FD0</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpeed_x1</sender>
   <signal>triggered()</signal>
   <receiver>actionSpeed_x1</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpeed_x2</sender>
   <signal>triggered()</signal>
   <receiver>actionSpeed_x2</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpeed_x4</sender>
   <signal>triggered()</signal>
   <receiver>actionSpeed_x4</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpeed_x8</sender>
   <signal>triggered()</signal>
   <receiver>actionSpeed_x8</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpeed_x16</sender>
   <signal>triggered()</signal>
   <receiver>actionSpeed_x16</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEject_FD0</sender>
   <signal>triggered()</signal>
   <receiver>actionEject_FD0</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action100ms</sender>
   <signal>triggered()</signal>
   <receiver>action100ms</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action200ms</sender>
   <signal>triggered()</signal>
   <receiver>action200ms</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action300ms</sender>
   <signal>triggered()</signal>
   <receiver>action300ms</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action400ms</sender>
   <signal>triggered()</signal>
   <receiver>action400ms</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action50ms</sender>
   <signal>triggered()</signal>
   <receiver>action50ms</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action2000Hz</sender>
   <signal>triggered()</signal>
   <receiver>action2000Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action4000Hz</sender>
   <signal>triggered()</signal>
   <receiver>action4000Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action8000Hz</sender>
   <signal>triggered()</signal>
   <receiver>action8000Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action11025Hz</sender>
   <signal>triggered()</signal>
   <receiver>action11025Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action22050Hz</sender>
   <signal>triggered()</signal>
   <receiver>action22050Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action44100Hz</sender>
   <signal>triggered()</signal>
   <receiver>action44100Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action48000Hz</sender>
   <signal>triggered()</signal>
   <receiver>action48000Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action96000Hz</sender>
   <signal>triggered()</signal>
   <receiver>action96000Hz</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>destroyed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
